news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md

```diff
@@ -0,0 +1,93 @@
+- [39 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-03-18..2019-03-24) in the project repositories;
+- [43 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-03-18..2019-03-24), including [18 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-03-18..2019-03-24).
+
+
+## Code changes in the 'develop' branch (for v1.7.6.0)
+
+### Core
+
+* [#9337](https://github.com/PrestaShop/PrestaShop/pull/9337): Migrate "Brands" listing page (a.k.a manufacturers). Thank you [@sarjon](https://github.com/sarjon)
+* [#11962](https://github.com/PrestaShop/PrestaShop/pull/11962): Do not stripslashes value from _GET & _POST, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#12940](https://github.com/PrestaShop/PrestaShop/pull/12940): Do not look for translations if no locale, by [@matks](https://github.com/matks)
+* [#12956](https://github.com/PrestaShop/PrestaShop/pull/12956): Customization - improve phpdoc and constants usage. Thank you [@Matt75](https://github.com/Matt75)
+* [#12959](https://github.com/PrestaShop/PrestaShop/pull/12959): Handle Tools curl timeout to ease debugging, by [@matks](https://github.com/matks)
+* [#12960](https://github.com/PrestaShop/PrestaShop/pull/12960): DoctrineBuilderExtension uses YamlParser for its config, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Back office
+
+* [#12641](https://github.com/PrestaShop/PrestaShop/pull/12641): Tax create/edit form migration. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#12647](https://github.com/PrestaShop/PrestaShop/pull/12647): Migration of "Design -> pages -> add new / edit page category". Thank you [@tomas862](https://github.com/tomas862)
+* [#12705](https://github.com/PrestaShop/PrestaShop/pull/12705): Update OrderHistory to manage {followup} & {shipping_number} vars in all email templates, not only in in_transit template. Thank you [@jf-viguier](https://github.com/jf-viguier)
+* [#12782](https://github.com/PrestaShop/PrestaShop/pull/12782): Fixes for migrated Categories add/edit actions. Thank you [@sarjon](https://github.com/sarjon)
+* [#12897](https://github.com/PrestaShop/PrestaShop/pull/12897): Enable lite display for new theme, by [@matks](https://github.com/matks)
+* [#12946](https://github.com/PrestaShop/PrestaShop/pull/12946): Fix issue in AdminCarrierWizardController::processRanges(). Thank you [@Matt75](https://github.com/Matt75)
+* [#12971](https://github.com/PrestaShop/PrestaShop/pull/12971): Enable customers page. Thank you [@sarjon](https://github.com/sarjon)$
```

Comment:
There's a `$` sign at the end that is useless 😉

Reply:
done, thanks